examples/rust/Earthfile

```diff
@@ -0,0 +1,44 @@
+FROM rust:1.49
+WORKDIR /example-rust
+
+plan:
+    RUN cargo install cargo-chef --version 0.1.11
+    
+    # Compute a lock-like file for our project
+    RUN cargo chef prepare --recipe-path recipe.json
+    SAVE ARTIFACT recipe.json recipe.json
+deps:
+    FROM +plan
+
+    COPY . .
+    COPY +plan/recipe.json .
+
+    RUN cargo chef cook --release --recipe-path recipe.json
+    SAVE ARTIFACT /usr/local/cargo cargo
+    SAVE ARTIFACT target target
+
+build:
+    FROM +deps
+
+    # Copy over the cached dependencies
+    COPY +deps/target .
+    COPY +deps/cargo /usr/local/cargo
+
+    ENV PORT 9091
+    RUN cargo build --release --bin example-rust
+    
+    SAVE ARTIFACT target/release/example-rust example-rust
+
+docker:
+    FROM debian:buster-slim
+    
+    COPY +build/example-rust example-rust
```

Comment:
It might be better to move this copy below the `apt-get` calls so that it conserves as much cache as possible between runs.

Reply:
Sure thing.